tests/main/interfaces-daemon-notify/task.yaml

```diff
@@ -21,14 +19,15 @@ execute: |
     echo "The interface is not connected by default"
     snap interfaces -i daemon-notify | MATCH -- "- +test-snapd-daemon-notify:daemon-notify"
 
+    echo "The service cannot be started with the interface disconnected"
+    snap start --enable test-snapd-daemon-notify.notify
```

Comment:
Nitpick: I think `start` is sufficient, `--enable` is not needed since we are not trying to enable the unit to be active across reboots.

Reply:
Done